container/worker/run_openqa_worker.sh

```diff
@@ -9,8 +9,15 @@ fi
 /root/qemu/kvm-mknod.sh
 
 if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  group=$(ls -lhn /dev/kvm | cut -d ' ' -f 4)
+  groupmod -g "$group" --non-unique kvm
+  [[ $(getent group kvm | cut -d ":" -f3) -ne "$group" ]] && { echo "Unable to ensure the ID of the kvm matches the group ID of /dev/kvm. KVM will not work."; exit 1; } 
```

Comment:
Didn't we come to the conclusion that this line is useless because it basically checks whether the previous command succeeds but with `set -e` it won't be executed anyways?